pages/chapters/[name].tsx

```diff
@@ -261,6 +267,7 @@ export async function getStaticProps(context: GetStaticPropsContext) {
             officers: JSON.parse(JSON.stringify(officers)) as Officer[],
             resources: JSON.parse(JSON.stringify(resources)) as Resource[],
         },
+        revalidate: 15,
```

Comment:
Can we extract the revalidate params to the urls or config file in utils? I put them in the urls file for now but we can have a separate place for globals from the urls.